### DISCUSSION TITLE
expand the storage pool

### DISCUSSION THREAD

**Discussion by obygyv** [007revad/Synology_enable_M2_volume#171] [Q&A] [answered]
How to expand the existing Storage Pool of the Synology with a M.2 disk? I would like to benefit from the atomic moves and the hard links between the same volume. With DS423+ and your script, I can't expand the existing storage pool (Volume1). I can only create a new one.

**Comment by 007revad** [comment, marked as answer]
Unfortunately DSM won't let you mix drive types in the same storage pool.
